core-services/image-mirroring/_config.yaml

```diff
@@ -519,6 +519,8 @@ supplementalCIImages:
     image: quay.io/powercloud/powervs-actions-container:latest
   ocp-on-powervs/powervs-container-host:powervs-ocp-jenkins-deployment:
     image: quay.io/powercloud/powervs-ocp-jenkins-deployment:latest
+  odh-multi/docker-dind:latest:
```

Comment:
whats the source for this image? I'm wondering if we can push the container image to `quay.io/powercloud` with instructions how to build.

Reply:
It's the official docker image just mirrored to my personal quay.io account. It would be better if it can be pushed to powercloud.

Reply:
in that case, why not directly mirror from the original location instead of mirroring to person repo? 🤔

Reply:
OpenShift CI cannot pull from `docker.io` due to pull rate limit. 
`We cannot mirror images from docker.io due to rate limiting constraints. Please, instead, push up an image to quay and mirror that to QCI.`

Reply:
As discussed lets move of one of the org repo and use that instead.